[expo-permissions] Update docs (#6699)
expo/expo: docs/pages/versions/unversioned/sdk/permissions.md

For [managed](../../introduction/managed-vs-bare/#managed-workflow) apps, you'll need to run `expo install expo-permissions`. To use it in a [bare](../../introduction/managed-vs-bare/#bare-workflow) React Native app, follow its [installation instructions](https://github.com/expo/expo/tree/master/packages/expo-permissions).

> ### Important Note (**Bare workflow only**)

Reviewer: ```suggestion
### Usage in bare workflow
```

> `expo-permissions` doesn't contain code responsible for requesting the system permission. To use this API, you need to install other expo packages, which contain requesters. For example, if you want to use `CAMERA_ROLL`, you need to install `expo-image-picker` or `expo-media-library`.

Reviewer: ```suggestion
`expo-permissions` includes the shared infrastructure for handling system permissions, it does not include the code specific to particular permissions. For example, if you want to use the `CAMERA_ROLL` permission, you need to install `expo-image-picker` or `expo-media-library`.
```

> You can find the location of the needed requester in the table below.

Reviewer: ```suggestion
The following table shows you which permissions correspond to which packages.
```

| Permission type             | Packages                                  |
| --------------------------- | ----------------------------------------- |
| `NOTIFICATIONS`             | `expo-permissions`                        |
| `USER_FACING_NOTIFICATIONS` | `expo-permissions`                        |
| `LOCATION`                  | `expo-location`                           |

> | `CAMERA`                    | `expo-camera`, `expo-barecode-scanner`    |

Reviewer: ```suggestion
| `CAMERA`                    | `expo-camera`, `expo-barcode-scanner`    |
```
